tests/ctst/steps/azureArchive.ts

```diff
@@ -372,10 +405,13 @@ When('i run sorbetctl to retry failed restore for {string} location', async func
         --kafka-object-task-topic=${this.parameters.KafkaObjectTaskTopic} \
         --kafka-brokers ${this.parameters.KafkaHosts}`;
     try {
-        await util.promisify(exec)(command);
+        const result = await util.promisify(exec)(command);
+        process.stdout.write(`Running command: ${command}\n`);
+        process.stdout.write(`Sorbetctl command result: ${result.stdout}\n`);
```

Comment:
Maybe you want to put the first log before running the command, in case of error, it will ease the debugging to have access to it